### PR TITLE
backstage: rewrite the o-layout test to be less flakey

### PR DESCRIPTION
**Pull request by JakeChampion**
The changes applied at to increase the viewport to one we know will work and to use `waitUntil` (https://open-wc.org/docs/testing/helpers/#waituntil) to periodically poll the dom until the aria-current attribute has been updated

This should resolve https://github.com/Financial-Times/origami/issues/329

### CODE REVIEW

**Review by chee — COMMENTED** [review]
woooooooo
:clap: :clap: :clap: :clap: :clap: :clap: :clap: 
thank you jake champion

**Review by chee — APPROVED** [review]
woooooooo
clap clap clap clap clap clap clap
thank you jake champion

File: components/o-layout/test/layout.test.js [hunk 1]
```diff
@@ -47,23 +48,28 @@ describe('Layout', () => {
...
-						`<div class="o-layout__sidebar"><nav class="o-layout__navigation"><ol class="o-layout__unstyled-element"><li class="o-layout__unstyled-element o-layout__navigation-title"><a class="o-layout__unstyled-element" href="#this-is-a-h1" aria-current="location">This is a heading level 1</a></li><li class="o-layout__unstyled-element "><a class="o-layout__unstyled-element" href="#this-is-a-h2" aria-current="false">This is a heading level 2</a><ol><li><a class="o-layout__unstyled-element" href="#sub-heading-1" aria-current="false">Sub heading 1</a></li><li><a class="o-layout__unstyled-element" href="#sub-heading-1b" aria-current="false">Sub heading 1b</a></li><li><a class="o-layout__unstyled-element" href="#sub-heading-2" aria-current="false">Sub heading 2</a></li></ol></li><li class="o-layout__unstyled-element "><a class="o-layout__unstyled-element" href="#this-is-a-second-h2" aria-current="false">This is a second heading level 2</a><ol><li><a class="o-layout__unstyled-element" href="#sub-heading-a" aria-current="false">Sub heading a</a></li></ol></li></ol></nav></div>`
-					);
-					done();
-				} catch (error) {
-					done(error);
+			await waitUntil(
```
Comment (notlee) [inline components/o-layout/test/layout.test.js:58]: nice! way better than one 200ms timeout

File: presets/eslint-config-origami-component/config.js [hunk 2]
```diff
@@ -121,11 +120,20 @@ module.exports = {
...
 	},
+   "overrides": [
+      {
+        "files": ["test/**/*.js"],
+        "rules": {
+          "@lwc/lwc/no-async-await": "off"
+        }
+      }
+    ]
 }
```
Comment (chee) [inline presets/eslint-config-origami-component/config.js:139]: this is fancy and cool

**Review by chee — APPROVED** [review]

**Pull request merged**